pkg/estimator/server/estimate.go

```diff
@@ -72,8 +73,6 @@ func (es *AccurateSchedulerEstimatorServer) estimateReplicas(
 
 func (es *AccurateSchedulerEstimatorServer) nodeMaxAvailableReplica(node *framework.NodeInfo, rl corev1.ResourceList) int32 {
 	rest := node.Allocatable.Clone().SubResource(node.Requested)
-	if rest.AllowedPodNumber > int64(len(node.Pods)) {
-		rest.AllowedPodNumber -= int64(len(node.Pods))
-	}
+	rest.AllowedPodNumber = util.MaxInt64(rest.AllowedPodNumber-int64(len(node.Pods)), 0)
```

Comment:
Does the pod number on the node not included in `node.Requested`?

Reply:
Remind me what the bug is?

Reply:
> Does the pod number on the node not included in node.Requested?

Pod number is included in `node.status`, but does not exist in `pod.spec`. So pod number should be handled specially.